src/startupscriptgenerator/src/common/parseUserCommands.go

```diff
@@ -0,0 +1,56 @@
+// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+
+package common
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+)
+
+func ParseUserRunCommand(sourcePath string) string {
+
+	appsvcFile, err := os.Open(sourcePath)
```

Comment:
Do we need to check for the existence of `sourcePath` first, or does `os.Open` do that for us?

Reply:
if the path does not exist `os.Open` returns a `pathError`